smarts/core/vehicle.py

```diff
@@ -89,13 +87,6 @@ def bbox(self) -> Polygon:
         )
         return shapely_rotate(poly, self.pose.heading, use_radians=True)
 
-    def __lt__(self, other):
-        """Allows ordering VehicleStates for use in sorted data-structures."""
-        assert isinstance(other, VehicleState)
-        return self.vehicle_id < other.vehicle_id or (
-            self.vehicle_id == other.vehicle_id and id(self) < id(other)
-        )
```

Comment:
This would appear to be a major change to how the vehicle states are compared, does this have to do with the ordering issues?

Reply:
This has just been moved to the parent class (`ActorState` in `actor.py`).  I think it should be inherited from there and have the same functionality as before.